### DISCUSSION TITLE
Does Ant Media Server supports for streaming HEVC (H.265) video to HTML5 Web players?

### DISCUSSION THREAD

**Discussion by asharamseervi** [ant-media/Ant-Media-Server#5095] [Q&A] [answered]
Does AMS enable support for HEVC (H.265) video to HTML5 web players, which can be critical in ultra-low bandwidth situations as it offers a much better compression ratio than H264/AVC? 
- How can I use AMS to get started with this?
- Are there any known limitations?

**Comment by lastpeony** [comment, marked as answer]
Hello please check this blog post on using H.265 on Ant Media Server
https://antmedia.io/the-first-hevc-h265-support-in-webrtc/
Its a great blog post with lots of information.
